docs/specs/openapi.json

```
@@ -5729,7 +5733,13 @@
                         "basic_auth": []
                     }]
                 }
-            }
+            },
+        "/recommendations/openshift": {
+            "$ref": "https://console.redhat.com/api/cost-management/v1/recommendations/openshift/openapi.json#/paths/~1recommendations~1openshift"
```

```suggestion
            "$ref": "/api/cost-management/v1/recommendations/openshift/openapi.json#/paths/~1recommendations~1openshift"
```
Don't put the domain. If we add the domain then this will only work in production and will fail in stage env.